src/main/java/org/jitsi/videobridge/EndpointConnectionStatus.java

```diff
@@ -210,41 +210,40 @@ private void doMonitor()
             for (Videobridge videobridge : jvbs)
             {
                 Conference[] conferences = videobridge.getConferences();
-                for (Conference conference : conferences)
-                {
-                    List<Endpoint> endpoints = conference.getEndpoints();
-                    for (Endpoint endpoint : endpoints)
-                    {
-                        monitorEndpointActivity(endpoint);
-                    }
-                }
+                Arrays.stream(conferences)
+                    .forEachOrdered(
+                        conference ->
+                            conference.getEndpoints()
+                                .forEach(this::monitorEndpointActivity));
 
                 cleanupExpiredEndpointsStatus();
             }
         }
     }
 
-    private void monitorEndpointActivity(Endpoint endpoint)
+    private void monitorEndpointActivity(EndpointBase endpointBase)
     {
+        if (!(endpointBase instanceof Endpoint))
+        {
+            // We only care about endpoints/participants connected to this
+            // bridge, which are of type Endpoint.
+            return;
+        }
+
+        Endpoint endpoint = (Endpoint) endpointBase;
         String endpointId = endpoint.getID();
-        long lastActivity = 0;
-        long mostRecentChannelCreated = 0;
 
         // Go over all RTP channels to get the latest timestamp
         List<RtpChannel> rtpChannels = endpoint.getChannels(null);
```

Comment:
this is another place where a method that takes no arguments (and conveys what that means) would be useful.

Reply:
I actually added the method already, but I didn't see this usage. Will update